styles/input-search.scss

```diff
@@ -0,0 +1,10 @@
+.c-input-search {
+	height: 56px;
+	display: inline-block;
+	box-sizing: border-box;
+	font: normal 400 18px/1 'Montserrat';
```

Comment:
please use a variable for font-family. Check all your files.
`font: normal 400 18px/1 $font-primary;`